stores/coffeeShopsStore.ts

```diff
@@ -11,7 +11,7 @@ interface CoffeeShopsState {
 const useCoffeeShopsStore = create<CoffeeShopsState>()(
   devtools(
     (set) => ({
-      coffeeShops: [],
```

Comment:
This was a bad initial state. This is always expected to be an object, so its default state should reflect that.